guide/additional-info/changes-in-v13.md

```diff
@@ -445,6 +495,18 @@ This method has been removed, with functionality replaced by the new `Permission
 + channel.permissionOverwrites.create(user, { VIEW_CHANNEL: false });
 ```
 
+#### GuildChannel#createInvite
+
+#### GuildChannel#fetchInvites
+
+These methods have been removed from `GuildChannel` and placed only on subclasses for which invites can be created. These are `TextChannel`, `NewsChannel`, `VoiceChannel`, `StageChannel` and `StoreChannel`.
+
+On these subclasses, the method now supports additional options:
+
+- `targetUser` to target the invite to join a particular streaming user
+- `targetApplication` to target the invite to a particular Discord activity
+- `targetType`
```

Comment:
This should probably be explained, like the other two fields are.

Reply:
It possibly could be "a type that grants additional features to the invite, 1 for a streaming user, 2 for voice channel activities..."?

Reply:
I mean isn't the type just specifying which of the above two it is?

Reply:
Then it should clarify that by describing it.